homeassistant/components/metoffice/weather.py

```diff
@@ -59,6 +60,9 @@ def _build_forecast_data(timestep):
         data[ATTR_FORECAST_WIND_BEARING] = timestep.wind_direction.value
     if timestep.wind_speed:
         data[ATTR_FORECAST_WIND_SPEED] = timestep.wind_speed.value
+    if not use_3hourly:
+        # if it's close to noon, mark as Day, otherwise as Night
+        data[ATTR_FORECAST_DAYTIME] = abs(timestep.date.hour - 12) < 6
```

Comment:
Daytime is not a standard item in the weather forecast. Please remove this. Here are the valid items:
https://developers.home-assistant.io/docs/core/entity/weather#forecast

Reply:
Huh, it is handled by frontend and is apparently used in other integrations: https://github.com/home-assistant/core/pull/52940#issuecomment-881428583
Removing it would mean reverting this PR entirely.

Shouldn't we consider "legalizing" it instead?

Reply:
That requires approval in an architecture discussion as noted in the frontend PR:
https://github.com/home-assistant/frontend/pull/6065#issuecomment-698361182

https://github.com/home-assistant/architecture/issues/382

Please revert this change. Thanks!

Reply:
Revert PR: https://github.com/home-assistant/core/pull/60978